src/main/resources/META-INF/plugin.xml

```diff
@@ -11,6 +11,7 @@
     ]]></description>
 
     <change-notes><![CDATA[
+      Version 0.6.0 - Improve stability with IntelliJ 2019.3.3+, Subversion support.
```

Comment:
I noticed the line breaks for the change notes do not carry though to the jetbrains notes.  Do other like plugins include HTML based breaks ?

Reply:
Yes, release notes have html formatting according to https://intellij-support.jetbrains.com/hc/en-us/community/posts/206752065-Recent-change-notes-formatting-

I will add \<br>